clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/view/VeniceViewWriter.java

```diff
@@ -62,18 +62,19 @@ public VeniceViewWriter(
    * @param newValue the incoming fully specified value which hasn't yet been committed to Venice
```

Comment:
Let's update the javadoc a bit here as newValueProvider and newValue sound similar but are certainly different.